main.js

```diff
@@ -9,18 +9,32 @@ define(function (require, exports, module) {
         LiveDevServerManager = brackets.getModule("LiveDevelopment/LiveDevServerManager"),
         PreferencesManager   = brackets.getModule("preferences/PreferencesManager"),
         ProjectManager       = brackets.getModule("project/ProjectManager"),
+        LiveDevelopment      = brackets.getModule("LiveDevelopment/LiveDevMultiBrowser"),
+        // PostMessageTransport = require("LiveDevelopment/MultiBrowserImpl/transports/PostMessageTransport"),
         NoHostServer         = require("nohost/src/NoHostServer").NoHostServer;
-    
+
     function _createServer() {
         var config = {
             pathResolver    : ProjectManager.makeProjectRelativeIfPossible,
             root            : ProjectManager.getProjectRoot().fullPath
         };
-        
+
         return new NoHostServer(config);
     }
 
-    AppInit.appReady(function () {
+    function _configureIframeForUse() {
+        // Set transport
+            // PostMessageTransport.setIframe(browser.getIframe());
+            // LiveDevelopment.setTransport(PostMessageTransport);
+    }
+    LiveDevelopment.one("statusChange", _configureIframeForUse);
```

Comment:
Add some documentation above this to explain the timing of what we're doing here.